Flickr/Flickr/ImageViewController.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  ImageViewController.swift
+//  Flickr
+//
+//  Created by metoSimka on 25/02/2019.
+//  Copyright © 2019 metoSimka. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+import collection_view_layouts
+
+class ImageViewController: UIViewController, ContentDynamicLayoutDelegate, UICollectionViewDelegate, UICollectionViewDataSource, UITextFieldDelegate {
+    
+    @IBOutlet weak var collectionView: UICollectionView!
+    @IBOutlet weak var searchTextField: UITextField! {
+        didSet {
+            searchTextField.delegate = self
+        }
+    }
+    @IBInspectable
+    var maxImagesToShow = 50 // just thinking about this, may be delete this idea. //sd
+    var imageList = [[String: AnyObject]]()
```

Comment:
need to make separate model class for containing image data instead of [String: AnyObject]